rootfs/scheduler/__init__.py

```diff
@@ -427,6 +427,9 @@ def _update_application_service(self, namespace, name, app_type, image, routable
             # Find if target port exists already, update / create as required
             if routable:
                 port = self._get_port(image)
+                if port is None:
```

Comment:
so if the app does not expose a port, we still want to expose port 5000? I think I'm missing something here. :confused: